packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -53,9 +57,17 @@ class ComplianceRemediationButton extends React.Component {
         });
     }
 
-    noRemediationAvailable = () => {
+    remediationAvailable = () => {
         const { allSystems, selectedRules } = this.props;
-        return selectedRules.length === 0 && !allSystems.some((system) => system.ruleObjectsFailed.some((rule) => rule.remediationAvailable && system.supported));
```

Comment:
Note here that `system.supported` is always `undefined`. `supported` is an attribute on `profile`.